src/Application/ZeitBuchung.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace ZeitBuchung\Application;
+
+
+use Symfony\Component\Console\Application;
+use ZeitBuchung\Command\Report;
+use ZeitBuchung\Command\Start;
+use ZeitBuchung\Command\Status;
+use ZeitBuchung\Command\Stop;
+
+class ZeitBuchung extends Application
+{
+
+    public function __construct(string $name = 'UNKNOWN', string $version = 'UNKNOWN')
+    {
+        parent::__construct($name, $version);
+        $this->add(new Start());
+        $this->add(new Stop());
+        $this->add(new Report());
+        $this->add(new Status());
+    }
+
+}
```

Comment:
PSR-12:
Line 24 | ERROR | [x] The closing brace for the class must go on the next line after the body
Line 24 | ERROR | [x] Expected 1 newline at end of file; 0 found
Line 24 | ERROR | [x] Whitespace found at end of line